honeybee_vtk/assistant.py

```diff
@@ -196,8 +209,8 @@ def auto_text_height(self, image_width, image_height) -> None:
     def _export_image(
             self, folder: str, image_type: ImageTypes = ImageTypes.png, *,
             image_scale: int = 1, image_width: int = 0, image_height: int = 0,
-            color_range: vtk.vtkLookupTable = None, rgba: bool = False,
-            show: bool = False) -> str:
+            image_name: str = '',  color_range: vtk.vtkLookupTable = None,
```

Comment:
Setting the default to '' is not a good idea. Set them to None.

Reply:
Yes. this is my bad.